sql/src/test/java/io/crate/integrationtests/InformationSchemaTest.java

```diff
@@ -1062,8 +1062,8 @@ public void testPartitionsNestedCol() {
         assertEquals(2, response.rowCount());
 
         assertThat(printedTable(response.rows()),
-            is("my_table| 04130| {metadata['date']=0}\n" +
-               "my_table| 04732d1g64p36d9i60o30c1g| {metadata['date']=1401235200000}\n"));
+            is("my_table| 045j2e9n60mj0c9d60og| {metadata['date']=0}\n" +
```

Comment:
Why did this change here?

Reply:
It is due the way we create process partitioned values. Previously, we did it in the `InsertFromValuesAnalyzer` https://github.com/crate/crate/blob/b9dbbd97f7e441188ddb24f4155e6688642d660f/sql/src/main/java/io/crate/analyze/InsertFromValuesAnalyzer.java#L512 Now, it is handled with `IndexNameResolver` https://github.com/crate/crate/blob/master/sql/src/main/java/io/crate/execution/engine/indexing/IndexNameResolver.java#L46. It seems that it calculates the value a bit different.

Reply:
This seems like a breaking change - potentially also a bug?

Reply:
@mfussenegger yep, seems like an issue if a table is partitioned by an object fields of the timestamp type. We create partitioned values with `PartitionName#encodeIdent`. In the case of the insert from subquery a timestamp is represented as string, in case of the insert from values - long, that leads to creation of the different partition names for the same timestamp value. Seems like there is a bug in `InputColumns#tryCreateSubscriptOnRoot` and the insert from subquery produces not exactly correct partition values.

Reply:
https://github.com/crate/crate/pull/9546